lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -346,7 +346,10 @@ public static function get_user_custom_post_type_id() {
 	 */
 	public static function theme_has_support() {
 		if ( ! isset( self::$theme_has_support ) ) {
-			self::$theme_has_support = is_readable( get_theme_file_path( 'theme.json' ) );
+			self::$theme_has_support = (
```

Comment:
Could `self::get_file_path_from_theme()` be used here? Not a big deal, but it would cut down on a small amount of duplicate code. Something like:
```
(
    is_readable( self::get_file_path_from_theme( 'theme.json' ) ) || 
    is_readable( self::get_file_path_from_theme( 'theme.json', true ) )
)
```

Reply:
Updated in 8cd92e4.